front-end/src/components/Absence/Absence.vue

```diff
@@ -465,7 +573,7 @@ export default {
         .catch(() => {});
     },
     handleAccept() {
-      patchAbsence(this.editAbsence)
+      patchAbsence(this.editAbsence.id, this.editAbsence)
         .then(() => {
```

Comment:
Can you also change all `then`/`catch` expressions to `async`/`await` in this file, please?

Reply:
done.